Modules/Components/Niftyreg/include/selxNiftyregf3dComponent.hxx

```diff
@@ -206,14 +206,13 @@ Niftyregf3dComponent<  TPixel >
       }
       else
       {
-        std::cout << "The key " << criterion.first << " is recognized, but not with value " << criterion.second[ 0 ] << std::endl;
+        this->Warning( "Criterion {0} has invalid value.", criterion.first );
```

Comment:
Why is this message changed? I think it is informative to tell the user what setting was read from the configuration (or multiple) if that gives producess the error. Why make it less informative?